ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh

```diff
@@ -36,7 +36,7 @@ function populate_artifact_dir() {
   # Copy CAPI-generated artifacts if they exist
   if [ -d "${dir}/.clusterapi_output" ]; then
     echo "Copying Cluster API generated manifests..."
-    cp -rpv "${dir}/.clusterapi_output" "${ARTIFACT_DIR}/" 2>/dev/null
+    cp -rpv "${dir}/.clusterapi_output/"{,**/}*.{log,yaml} "${ARTIFACT_DIR}/" 2>/dev/null
```

Comment:
Would you be against
```suggestion
    cp -rpv "${dir}/.clusterapi_output/"{,**/}*.{log,yaml} "${ARTIFACT_DIR}/clusterapi_output" 2>/dev/null
```

The shell-fu [is working](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/53386/rehearse-53386-pull-ci-openshift-installer-master-altinfra-e2e-aws-ovn/1803128670945021952/artifacts/e2e-aws-ovn/ipi-install-install/artifacts/) but I think I preferred when the capi output was inside its own directory.

Reply:
(I think you'd have to `mkdir -p ${ARTIFACTS_DIR}/clusterapi_output` first, but you get the idea).

Reply:
> Would you be against

> I preferred when the capi output was inside its own directory.

No, it was my intent as well to keep this stuff in its own dir. incorporated changes into the most recent push and running a rehearsal.